src/main/java/duke/command/TaskManager.java

```diff
@@ -13,6 +19,7 @@ public TaskManager() {
      *
      * @param description: user task to remember
      */
+
     public void addToList(String description) {
         Task task = new Task(description);
```

Comment:
Inconsistent naming of variable, new task is named as "task" while for new deadlines and new event it is named as "event".

Reply:
Oops my bad forgot to edit after copy pasting